### PR TITLE
Convert SCSS Variables into CSS Custom Properties/Variables

### PR DESCRIPTION
**Pull request by mAiNiNfEcTiOn**
# What does this PR do:

* Changes the `builder` to use the new processor `cssvars` to generate the `dist/theme.css` with a general declaration of all variables.

* Changes the `components/**/*.scss` files to use CSS variables instead of SCSS variables.

* Changes the `styleguide.html` to include the `theme.css` at the `<head>`.

Proof

# Where should the reviewer start:
  - Diffs
  - Check your component's styles
  - Check the styleguide

# Unit and/or functional tests:
One test fixed, all tests are passing.

### CODE REVIEW

**Comment by fahad19** [comment]
💖 💯 👍 🎉

File: components/autoComplete/autoComplete.scss [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
-  font-family: $tx-generic-font-primary-font-family, $tx-generic-font-primary-generic-family;
+  font-family: var(--tx-generic-font-primary-font-family), var(--tx-generic-font-primary-generic-family);
   position: relative;
 }
 
 .ui-autocomplete__popunder {
-  background-color: $tx-autocomplete-background-color;
-  box-shadow: 0 0 5px 0 rgba($tx-autocomplete-box-shadow, .18);
+  background-color: var(--tx-autocomplete-background-color);
+  box-shadow: var(--tx-autocomplete-box-shadow);
```
Comment (AlexDudar) [inline components/autoComplete/autoComplete.scss]: did you extend `tx-autocomplete-box-shadow` or we don't need `0 0 5px 0` anymore?

File: components/dropDown/dropDown.scss [hunk 2]
```diff
@@ -29,13 +17,13 @@ $select-arrow-color: $tx-dropdown-arrow-color !default;
 
   .Select-menu-outer {
     border: none;
-    box-shadow: 0 0 5px 0 rgba($tx-dropdown-select-menu-outer-box-shadow, .18);
```
Comment (AlexDudar) [inline components/dropDown/dropDown.scss]: same question here. why did you remove `0 0 5px 0 `?

File: components/autoComplete/autoComplete.scss (hunk 1 above)
Comment (AlexDudar) [inline components/autoComplete/autoComplete.scss]: i found an answer :)

Comment (mAiNiNfEcTiOn) [inline components/autoComplete/autoComplete.scss]: The value changed to the variable. Before it was complicated to do it with SASS vars, but with CSS vars they're properly interpreted.

https://github.com/Travix-International/travix-ui-kit/pull/99/files#diff-c7cd1d0bc3ebfea88dd553f8f9d24d8aR76
